newIDE/app/src/UI/Messages/DismissableInfoBar.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+import { Trans } from '@lingui/macro';
+import * as React from 'react';
+import Snackbar from '@material-ui/core/Snackbar';
+import Button from '@material-ui/core/Button';
+import PreferencesContext, {
+  type AlertMessageIdentifier,
+} from '../../MainFrame/Preferences/PreferencesContext';
+import { useScreenType } from '../Reponsive/ScreenTypeMeasurer';
+
+type Props = {|
+  identifier: AlertMessageIdentifier,
+  message: React.Node,
+  touchScreenMessage?: React.Node,
+  show: boolean,
+|};
+
+const DismissableInfoBar = ({
+  identifier,
+  show,
+  touchScreenMessage,
+  message,
+}: Props) => {
+  const preferencesContext = React.useContext(PreferencesContext);
```

Comment:
To rename `preferences` for consistency's sake

Reply:
Correct, context does not appear in the variable name (much like we don't call a `useState` variable `somethingState` and `setSomethingState` but just `something/setSomething`).